tiledb/sm/query/result_tile.cc

```diff
@@ -1133,19 +934,18 @@ void ResultTile::compute_results_bitmap_sparse(
   Buffer* const buffer = coords_tile.buffer();
   const T* const coords = static_cast<const T*>(buffer->data());
   {
-    if (has_previous_dim) {
-      for (uint64_t pos = 0; pos < coords_num; ++pos) {
-        c = coords[pos * dim_num + dim_idx];
-        if (c >= r0 && c <= r1)
-          r_bitmap[pos] &= 1;
-        else
-          r_bitmap[pos] = 0;
-      }
-    } else {
-      for (uint64_t pos = 0; pos < coords_num; ++pos) {
-        c = coords[pos * dim_num + dim_idx];
-        if (c >= r0 && c <= r1)
-          r_bitmap[pos] = 1;
+    for (uint64_t pos = 0; pos < coords_num; ++pos) {
```

Comment:
I think this should focus only on operators `OR` and `AND` instead of `+` and `*`.

Reply:
Those are required for multiplicities of result...